### ISSUE TITLE
[eqeqeq] auto fixer breaks code

### ISSUE THREAD

**Issue by mattlewis92** [angular-eslint/angular-eslint#538]
**Description and reproduction of the issue**

```JSON
{
  "rules": {
    "@angular-eslint/template/eqeqeq": ["warn"]
  }
}
```

```HTML
<span
  *ngIf="
    mode == 'list'
  "
  >+</span
>

```

After `eslint --fix` is ran:

```HTML
<span
  *ngIf="
    mode == 'lis==========t'
  "
  >+</span
>

```

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `12.1.0` |
| `@angular-eslint/template-parser`         | `12.1.0` |
| `@typescript-eslint/parser`               | `12.1.0` |
| `ESLint`                                  | `7.28.0` |
| `node`                                    | `14.17.0` |

```sh
Angular CLI: 12.0.4
Node: 14.17.0
Package Manager: npm 6.14.13
OS: darwin x64

Angular: 12.0.4
... animations, cdk, cli, common, compiler, compiler-cli, core
... elements, forms, google-maps, language-service, localize
... platform-browser, platform-browser-dynamic, router
... service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.4
@angular-devkit/build-angular   12.0.4
@angular-devkit/core            12.0.4
@angular-devkit/schematics      12.0.4
@schematics/angular             12.0.4
ng-packagr                      12.0.2
rxjs                            6.6.3
typescript                      4.2.4

```

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by rafaelss95** [comment]
The problem itself is not in the fixer, but rather in the report... it currently affect many, if not all, rules, as reported in #384 for example.

This excerpt

```html
<span
  *ngIf="
    mode == 'list'
  "
  >+</span
>
```

should report

```json
{
  "start": { 
    "line": 3, 
    "column": 5
  }, 
  "end": { 
    "line": 3, 
    "column": 19
  }
}
```

but it's reporting

```json
{
  "start": { 
    "line": 3, 
    "column": 10
  }, 
  "end": { 
    "line": 5, 
    "column": 1
  }
}
```

**Comment by ArtemFrontendACTUM** [comment]
Same issue here

**Comment by stevebor1** [comment]
Just upgraded from Angular 10 through 14 and ended up with a ton of botched linting. Many of these type of clean-ups after the upgrade. `mode == 'lis==========t`

**Comment by yaronuliel** [comment]
Same here - fresh installation with versions:


package | version
-- | --
@angular-eslint/eslint-plugin-template | 16.1.2
@angular-eslint/template-parser | 16.1.2
@typescript-eslint/parser | 6.7.0
ESLint | 8.49.0
node | 18.17.1

**Comment by ajitzero** [comment]
Same issue. Disabled this rule for *.html for now.

Do we have any leads on this?

**Comment by mattlewis92** [comment]
Closing as this was fixed in `17.3.0` 🥳
